app/bundles/ReportBundle/Tests/Model/ReportModelTest.php

```diff
@@ -34,6 +34,24 @@ protected function setUp(): void
         $fieldModelMock = $this->createMock(FieldModel::class);
         $fieldModelMock->method('getPublishedFieldArrays')->willReturn([]);
 
+        $this->reportModel = new ReportModel(
```

Comment:
Why is the reportModel created here again when it's created on line 70 again?

Reply:
It was added during merging. I have removed